packages/arb-rpc-node/batcher/sequencerBatcher.go

```diff
@@ -268,6 +268,10 @@ func txLogsToResults(logs []core.ValueAndInbox) (map[common.Hash]*evm.TxResult,
 const maxTxDataSize int = 100_000
 
 func (b *SequencerBatcher) SendTransaction(startCtx context.Context, startTx *types.Transaction) error {
+	if b.config.Node.Sequencer.Dangerous.DisableUserMessageSequencing {
+		return errors.New("user transaction sequencing disabled")
```

Comment:
What should this error be? "Arbitrum is temporarily unavailable while migrating to Nitro"?